fiftyone/utils/github.py

```
@@ -45,8 +49,41 @@ def __init__(self, repo):
+            if "/" not in ref:
+                urls.append(f"{api_root}/commits/{ref}")  # commit
+
+            for url in urls:
+                try:
+                    _ = self._get(url)
+                    if ref != self._ref:
+                        self._ref = ref
+                        self._safe_path = "/".join(chunks[i:])
+
+                    return
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)